### ISSUE TITLE
Startseite: Logo Verbund der offenen Werkstätten zu klein

### ISSUE THREAD

**Issue by Merlind** [fairmondo/fairmondo#1398]
Ist es möglich, das Logo irgendwie größer anzuzeigen? Man kann die Schrift ja überhaupt nicht lesen. Dafür müssten wir eine andere Lösung finden. Kann man die Logos unterschiedlich lang oder breit anzeigen? 
LG Merlind

**Comment by marcaltmann** [comment]
Ja. Im Moment werden ja einfach die Nutzerbilder von den Marktplatz-Konten der Organisationen genommen. Man kann das schöner machen, indem man die Grafiken separat speichert.

**Comment by marcaltmann** [comment]
It would be interesting to create a new solution for all the references/partners/sponsors images at the bottom of the welcome page. One that is fully based on CSS sprites, looks nicer and maybe is somehow optically attached to the footer.
